### PR TITLE
Fix text in english mobile banner

### PR DESCRIPTION
**Pull request by tzhelyazkova**
:pencil2: :spiral_notepad:

### CODE REVIEW

File: mobile_english/banner_ctrl.js [hunk 1]
```diff
@@ -64,13 +64,16 @@ const progressBarTextInnerLeft = [
 	numberOfDaysUntilCampaignEnd > 1 ? Translations[ 'day-plural' ] : Translations[ 'day-singular' ],
 	Translations[ 'suffix-days-left' ]
 ].join( ' ' );
-const progressBarTextRight = 'Still missing: <span class="js-value_remaining">1,2</span> Mio. €';
+const progressBarTextRight = 'Still missing: € <span class="js-value_remaining">1,2</span> Mio.';
```
Comment (ghost) [inline mobile_english/banner_ctrl.js:67]: Still a comma here.

File: mobile_english/banner_var.js [hunk 2]
```diff
@@ -59,7 +59,8 @@ const CampaignName = $bannerContainer.data( 'campaign-tracking' );
 const BannerName = $bannerContainer.data( 'tracking' );
 const ProgressBar = require( '../shared/progress_bar/progress_bar' );
 const numberOfDaysUntilCampaignEnd = campaignDays.getNumberOfDaysUntilCampaignEnd();
-const progressBarTextRight = 'Still missing: <span class="js-value_remaining">1,2</span> Mio. €';
+const progressBarTextRight = 'Still missing: € <span class="js-value_remaining">1,2</span> Mio.';
```
Comment (ghost) [inline mobile_english/banner_var.js:62]: Still a comma here.

**Review by ghost — CHANGES_REQUESTED** [review]
@KaiNissen The text still contains "Mio." as an abbreviation for "million", this is German and not a common abbreviation in English. "Million" is commonly abbreviated as simply "M" in English as far as I know.

Aside from that @tzhelyazkova: :heavy_check_mark: :us: :cowboy_hat_face: :+1: :us:

File: mobile_english/banner_ctrl.js (hunk 1 above)
Comment (tzhelyazkova) [reply mobile_english/banner_ctrl.js:67]: It's irrelevant here, the number is just a placeholder. The separator `.` is passed in the `options`.

Comment (ghost) [reply mobile_english/banner_ctrl.js:67]: e9f

**Review by ghost — APPROVED** [review]

File: mobile_english/banner_ctrl.js (hunk 1 above)
Comment (tzhelyazkova) [reply mobile_english/banner_ctrl.js:67]: :arrow_up: That's you in your message to @KaiNissen :nerd_face:

**Comment by KaiNissen** [comment]
I agree to what @timEulitz says: the banners should print "€ 2.3M".

**Pull request merged**